### ISSUE TITLE
RFE: make editor error icons open bottom pane on click for more details

### ISSUE THREAD

**Issue by james-lawrence** [joefitzgerald/go-plus#843]
### Prerequisites

* [x] Have you tried launching `atom .` from the terminal in your project's directory?
* [x] Have you verified the output from `go env` is correct? If it is, please include the output in this issue.
* [x] Have you updated Atom to the latest version?
* [ ] Have you tried using [Atom Beta](https://atom.io/beta), which can be run side-by-side with Atom Stable? Is the behavior the same, or different?
* [x] Have you updated your Atom packages to the latest versions?
* [x] Have you read the [FAQ](https://github.com/joefitzgerald/go-plus/wiki/FAQ)?
* [x] Have you searched [the issues](https://github.com/joefitzgerald/go-plus/issues?utf8=%E2%9C%93&q=is%3Aissue) to see if others are experiencing the same issue?

### Description

minor quibble but the errors in the side of the editor should be clickable to focus on them in the bottom pane, and defaults for the editor should likely be changed to prevent the bottom pane from showing automatically. provides a better UX.

### Output from `atom -v && apm -v`

```
Atom    : 1.32.2
Electron: 2.0.9
Chrome  : 61.0.3163.100
Node    : 8.9.3
apm  2.1.2
npm  6.2.0
node 8.9.3 x64
atom 1.32.2
python 2.7.15
git 2.19.2
```

### Expected Behavior

- by default errors while coding should be highlighted without causing the editor to reposition text.
- nice to have: sidebar details should be clickable to display bottom pane with the clicked info focused.

### Actual Behavior

by default bottom pane pops up on every minor error while working which is extremely jarring UX as it moves the text unexpectedly forcing the user to relocate their position.

### Related issues

#318 #425 #601

**Comment by zmb3** [comment]
You can disable the _Focus the go-plus panel on failure_ setting to prevent the go-plus pane from opening when something fails.

The errors in the editor and the diagnostics panel are actually provided by atom-ide-ui.  If it's the diagnostics panel that's bothering you, I would try disabling the  _Automatically show and hide the diagnostics panel_ setting under the atom-ide-ui package.

**Comment by james-lawrence** [comment]
right, I found those settings. I'm putting forth that the defaults need to be changed.
